client/components/Register.jsx

```diff
@@ -49,27 +58,55 @@ class Register extends React.Component {
         <br/>
         <label htmlFor='surname'>Surname</label>
         <br/>
-        <input type='text' placeholder='Last name' name='surname'
+        <input 
+          type='text' 
+          placeholder='Last name' 
+          name='surname'
           className='u-full-width'
-          value={this.state.name}
+          value={this.state.surname}
           onChange={this.handleChange}
         />
         <br/>
         <br/>
         <label htmlFor='hash'>Hash</label>
         <br/>
-        <input type='password' placeholder='Password' name='password'
+        <input 
+          type='password' 
+          placeholder='Password' 
+          name='password'
           className='u-full-width'
-          value={this.state.description}
+          value={this.state.password}
           onChange={this.handleChange}
         />
         <br/>
         <br/>
-        <label htmlFor='role'>Role</label>
+        <br/>
```

Comment:
Remove the `<br />` tags and use CSS to create the space you need. This should be done for this whole component. Using elements and attributes for styling is an anti-pattern. CSS is more than capable these days :wink: